tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -39,8 +39,7 @@ def task(event, context):    #pylint: disable-msg=unused-argument
     LOGGER.debug("event: {event}", event=event)
     try:
         config = event['config']
-        collection = config.get(CONFIG_COLLECTION_KEY, {})
-        exclude_file_types = collection.get(COLLECTION_META_KEY, {}).get(EXCLUDE_FILE_TYPES_KEY, [])
+        exclude_file_types = config['exclude_file_types']
```

Comment:
use EXCLUDE_FILE_TYPES_KEY like `exclude_file_types = config[EXCLUDE_FILE_TYPES_KEY]` instead of bare key here for consistency